components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/factories/ProductsApiServiceFactory.java

```diff
@@ -0,0 +1,14 @@
+package org.wso2.carbon.apimgt.rest.api.publisher.factories;
+
+import org.wso2.carbon.apimgt.rest.api.publisher.ProductsApiService;
+import org.wso2.carbon.apimgt.rest.api.publisher.impl.ProductsApiServiceImpl;
+
+public class ProductsApiServiceFactory {
```

Comment:
A class level comment would  give a better idea to others when reading the code.